src/main/java/com/users/service/UserService.java

```diff
@@ -0,0 +1,78 @@
+package com.users.service;
+
+import com.users.dto.UserDto;
+import com.users.entities.User;
+import com.users.exceptions.EntityAlreadyExistsException;
+import com.users.exceptions.EntityNotFoundException;
+import com.users.mappers.UserMapper;
+import com.users.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.Period;
+import java.time.ZoneId;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Transactional
+@AllArgsConstructor
+@Service
+@Slf4j
+public class UserService {
+    private UserRepository userRepository;
+    private UserMapper userMapper;
+
+    public UserDto create(User user) {
+        log.info("User to be created {}", user);
```

Comment:
de obicei pe metodele simple se foloseste .debug ca sa nu incarci consola cu multe loguri, se foloseste .info in punctele de intrarea gen controller, ca sa stii ca a intrat pe acolo si restu pe debug doar in caz ca vrei sa vezi ce se intampla sa muti pe debug
sa cauti putin pe net despre .debug .info .error si .warn

Reply:
adica pentru create read update delete, daca sunt simple pot renunta la log uri, nu?

Reply:
nu renunti, puis log.debug, in functie de nivelul de logare (info, debug, warn, error) si nivelul la care este setata aplicatia aceste log uri pot aparea sau nu in consola, cum metoda de create este o metoda simpluta care n-ar trebui sa dea batai de cap nu ai de ce sa loghezi ori de cate ori creezi un user

Reply:
ok, trebuie sa citesc putin asta